src/main/java/com/CezaryZal/api/body/manager/repo/BodySizeRepoService.java

```diff
@@ -0,0 +1,61 @@
+package com.CezaryZal.api.body.manager.repo;
+
+import com.CezaryZal.api.body.BodySizeRepository;
+import com.CezaryZal.api.body.entity.BodySize;
+import com.CezaryZal.exceptions.BodySizeNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.sql.Date;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class BodySizeRepoService {
+
+    private BodySizeRepository bodySizeR;
```

Comment:
zaczałeś tworzyć jakieś nazwy w stylu bodySizeR albo bodySizeS widziałeś to gdzieś w jakimś kodzie czy to Twój koncept?